common/hooks/post-install/06-strip-and-debug-pkgs.sh

```diff
@@ -72,7 +72,7 @@ hook() {
 
 		fname=${f##*/}
 		for x in ${nostrip_files}; do
-			if [ "$x" = "$fname" ]; then
+			if [ "$x" = "$fname" -o "$x" = "${f#$PKGDESTDIR}" ]; then
```

Comment:
I've not looked, but does this affect any other packages?

Reply:
There are 23 source packages using `nostrip_files`, they all of course use filename without path and that shouldn't be affected by this change. I don't know if there is another package with two files of the same name such that one is nostrip and one isn't -- probably not.